app/views/decidim/kids/user_minors/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<% content_for(:subtitle) { t("menu", scope: "decidim.kids.user") } %>
+<div class="row">
+  <h3 class="section flex-center"><%= t(".title") %></h3>
+  <div class="columns large-6 medium-10 medium-centered">
+    <div class="card">
+      <div class="card__content">
+        <%= decidim_form_for(@form, url: user_minors_path) do |form| %>
+          <%= form_required_explanation %>
+
+          <div class="field">
+            <%= form.text_field :name, class: "js-hashtags" %>
+          </div>
+
+          <div class="field">
+            <%= form.date_field :birthday, class: "js-hashtags" %>
+          </div>
+
+          <div class="field">
+            <%= form.email_field :email, class: "js-hashtags" %>
+          </div>
+
+          <div class="field">
+            <%= form.password_field :password %>
+          </div>
+
+          <div class="field">
+            <%= form.password_field :password_confirmation %>
+          </div>
```

Comment:
You can probably reuse a common view (called `_form.html.erb`) for the new.html.erb and the edit.html.erb with this fields